README.md

```diff
@@ -1,6 +1,32 @@
-# Develop
+# Installing
 
-Requires Python 3.8+
+## conda (release)
+To install the latest stable release
+
+```bash
+conda install -c hexrd -c anaconda -c conda-forge hexrdgui
```

Comment:
We might want to clarify somewhere around here that python 3.8+ is required.

Reply:
Yes, lets not remove the fact that we need python 3.8, unless we want to support multiple versions ...

Reply:
I just fixed that.